1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,7 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function (){this.age +1;},/* todo */
```

Comment:
Good effort! You just need a small change here to make it actually update the age (can you figure it out?). Let me know if not and I'll share how I would do it!